libs/components/ag-grid/src/lib/modules/ag-grid/cell-editors/cell-editor-text/cell-editor-text.component.ts

```diff
@@ -22,10 +25,12 @@ import { SkyCellEditorTextParams } from '../../types/cell-editor-text-params';
 export class SkyAgGridCellEditorTextComponent
   implements ICellEditorAngularComp
 {
-  public value: string;
```

Comment:
Removing public field is a breaking change.

Reply:
Same comment about the component not being public.